week-6/Homework/mandatory/2-quotegenerator/quotes.js

```diff
@@ -1,3 +1,24 @@
+
```

Comment:
Looks good, well done! There are issues with quote formatting, which you can fix by replacing the quotes in data from '’' to `'`.